DYN-7466: padding factor

ivaylo-matov opened the pull request:
### Purpose

Small PR related to [DYN-7466](https://jira.autodesk.com/browse/DYN-7466) and feedback from the last Archilizer Check-in regarding nodes/groups appearing too large when zoom-to-fit is applied.

I've added a `paddingFactor` of 1.5 to `WorkspaceViewModel.FitViewInternal`. This adjustment affects zoom-to-fit behavior for node or group selections in TuneUp and Graph Node Manager, as well as for the Zoom to Fit button in workspace.

I think keeping `paddingFactor` as a local variable is appropriate, and the value of 1.5 seems reasonable. However, please let me know if this should be changed.

**_Before   ---> ---> --->    After_**

![Zoom-Node](https://github.com/user-attachments/assets/be8ef51e-9d68-45c1-b5df-a8b48b13db36)

![Zoom-Group](https://github.com/user-attachments/assets/2aae270c-b2a7-444a-932a-799f11caf099)


### Declarations

Check these if you believe they are true

- [ ] The codebase is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated
- [ ] This PR contains no files larger than 50 MB

### Release Notes

Padding factor added to zoom-to-fit, affecting TuneUp, Graph Node Manager, and the Zoom to Fit button.

### Reviewers
@reddyashish 
@QilongTang 

### FYIs
@dnenov 
@Amoursol

reddyashish commented:
DynamoCoreWpfTests.NodeAutoCompleteSearchTests.NodeSuggestions_OutputPortBuiltInNode_AreCorrect is failing. Can you check it @ivaylo-matov

ivaylo-matov commented:
@reddyashish , here it passes every time

![image](https://github.com/user-attachments/assets/77c15678-35f2-40d4-bcf4-1c7e28aed06b)

ivaylo-matov commented:
Updates the padding factor to match the screenshots from [this](https://autodesk.slack.com/archives/CPJ5UQW0Z/p1729879067010159) Slack thread
![image](https://github.com/user-attachments/assets/567593cf-d4ae-4d71-bb4b-225f0d39a9c9)
 <img src="https://github.com/user-attachments/assets/567593cf-d4ae-4d71-bb4b-225f0d39a9c9" width="300">

QilongTang commented:
> I've added a paddingFactor of 1.5 

The actual value is 3.5?

ivaylo-matov commented:
> > I've added a paddingFactor of 1.5
> 
> The actual value is 3.5?

Yes, updated the value after Helena's [feedback](https://autodesk.slack.com/archives/CPJ5UQW0Z/p1729879067010159). Please see my comment above.

QilongTang commented:
> > > I've added a paddingFactor of 1.5
> > 
> > 
> > The actual value is 3.5?
> 
> Yes, I updated the value after Helena's [feedback](https://autodesk.slack.com/archives/CPJ5UQW0Z/p1729879067010159). Please see my comment above.

Thank you, can you also help me test if this affect the zoom to fit feature for geometry, or only the graph nodes?

ivaylo-matov commented:
It's only for graph nodes and groups. Zoom-to-fit on geometry fills in the whole space. 
Just to clarify... should I add a test ?

QilongTang commented:
> It's only for graph nodes and groups. Zoom-to-fit on geometry fills in the whole space. Just to clarify... should I add a test ?

I dont see that blocking such PR because zoom to fit is more a UI feature. But in the mean time, I think most consts in Dynamo are defined in DynamoConfiguration project, can you move it over there?

ivaylo-matov commented:
The constant is now in Dynamo.Configuration. Sorry for the delay.

QilongTang commented:
Re-running self serve on https://master-5.jenkins.autodesk.com/job/Dynamo/job/DynamoSelfServe/job/pullRequestValidation/16662/

QilongTang commented:
Tests all pass now, merging. FYI: @reddyashish @avidit
